### PR TITLE
Add release notes for 2020.3.1 Demeter

### PR DESCRIPTION
**Pull request by kasenvr**
Also updated release note dates to follow ISO 8601 spec.

### CODE REVIEW

File: docs/source/release-notes/2020-3-1-Demeter.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* Graphics: Fix handling of missing glTF blendshape target meshes. (#773)
+* API: Add `url` parameter to `Window.openWebBrowser`. (#778)
+* App: Add script that prompts a user to enter a display name on first run if one is not set. (#805)
+* Application: Add 'Valve Index' workaround to allow calibration for full body tracking. (#808)
+* App: 'Explore' app now references 'Location Name' when adding a location. (#814)
+
+#### Miscellaneous
+
+* Application: Fix build warnings. (#761)
+* Application: Set GL preference to 'GLVND' from 'LEGACY' then revert back to 'LEGACY'. (#763, #806)
```
Comment (JulianGro) [inline docs/source/release-notes/2020-3-1-Demeter.md:25]: It went from undefined to GLVND and then to LEGACY again. =P

File: docs/source/release-notes/2020-3-1-Demeter.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+This version of Vircadia is an update to 2020.3.0. It is compatible with the 2020.3.0-Demeter protocol. This version was released on October 24th, 2020.
+
+## Interface (Codename Athena)
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).
+
+#### General
+
+* Application: Update shortcut names to include the "Vircadia" in them. (#651)
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-1-Demeter.md:11]:
```suggestion
* Application: Update shortcut names to include "Vircadia" in them. (#651)
```

File: docs/source/release-notes/2020-3-1-Demeter.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+* UI: Add a checkbox for more details to messages logged in entity scripts such as the scriptURL and function with line numbers. (#669)
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-1-Demeter.md:12]:
```suggestion
* UI: Add a checkbox for more details to messages logged in entity scripts such as the script URL and function with line numbers. (#669)
```

File: docs/source/release-notes/2020-3-1-Demeter.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+[Iamus](https://github.com/kasenvr/iamus) Metaverse Server updates can be found [here](https://github.com/kasenvr/Iamus/commits/master).
+
+[Iamus Metaverse Dashboard](https://github.com/vircadia/project-iamus-dashboard) updates can be found [here](https://dashboard.vircadia.com/).
+
+## Vircadia Launcher (Codename Pantheon)
+
+[Vircadia's launcher](https://github.com/vircadia/pantheon-launcher) updates can be found [here](https://github.com/vircadia/pantheon-launcher/commits/master). Some changes are listed below. The launcher is now on experimental version 0.0.1s.
+
+* Fix launching of Interface while the launcher is running as an admin.
+* Fix sandbox closing if the launcher is closed.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-1-Demeter.md:49]:
```suggestion
* Fix Sandbox closing if the launcher is closed.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**